pkg/updater/service/datadog_agent.go

```diff
@@ -42,6 +42,10 @@ var (
 
 // SetupAgentUnits installs and starts the agent units
 func SetupAgentUnits() (err error) {
+	if updaterErr := setupUpdater(); updaterErr != nil {
```

Comment:
temporarily in the Agent setup until bootstrap of a self updating updater is built